### PR TITLE
replace caxes by dims

### PR DESCRIPTION
**Pull request by lazarusA**
- [ ] remove `caxes(x)` and replace by `dims(x)`
- [ ] arrays vs tuple ?, `dims` outputs a tuple and `caxes` an array
- [ ] fix tests

### CODE REVIEW

File: src/Cubes/Cubes.jl [hunk 1]
```diff
@@ -204,16 +204,19 @@ function DD.rebuild(A::YAXArray; data=parent(A), dims=DD.dims(A), metadata=DD.me
...
+#     #@show typeof(x)
+#     dims = map(enumerate(dimnames(x))) do a
+#         index, symbol = a
+#         values = YAXArrayBase.dimvals(x, index)
+#         DD.Dim{symbol}(values)
+#     end
+#     (dims... ,)
+# end
+
+caxes(x) = collect(DD.dims(x))
```
Comment (felixcremer) [inline src/Cubes/Cubes.jl:218]:
```suggestion
caxes(x::DD.AbstractDimArray) = collect(DD.dims(x))
```

**Review by meggart — APPROVED** [review]

**Comment by meggart** [comment]
Looks good to me. Does mapCube work on Rasters with this change?

**Comment by lazarusA** [comment]
it does work. But, we still have the function `caxes` (doing the work at the saving stage), ideally this one  should also be gone for good.

Another issue is that functions provided to `mapCube` also can be those acting on `Arrays`, hence Raster function don't work, but that another issue.

**Comment by meggart** [comment]
What is the status here? Can this be merged for now and we leave the remaining changes for later?

**Comment by lazarusA** [comment]
Merging this for now should be ok. I will not go into making those other changes anytime soon.

**Pull request merged**